db_scripts/mosip_certify/db.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE mosip_certify
+	ENCODING = 'UTF8' 
+	LC_COLLATE = 'en_US.UTF-8' 
+	LC_CTYPE = 'en_US.UTF-8' 
+	TABLESPACE = pg_default 
+	OWNER = postgres
+	TEMPLATE  = template0;
+
+COMMENT ON DATABASE mosip_idp IS 'e-Signet and certify related data is stored in this database';
```

Comment:
Lets not mention the e-Signet part, since we don't have a hard dependency

Reply:
removed